src/components/organisms/Header/header.scss

```diff
@@ -18,7 +18,7 @@
 
   &__title {
     display: inline-block;
-    text-transform: uppercase;
+    // text-transform: uppercase;
```

Comment:
Prefer deleting unused lines. If you want to go back later, check the git history.

Reply:
Alright, I'll remove the comment.